### PR TITLE
Fix card group rounding

### LINKED ISSUE #17203: Card Group: Borders are not correctly un-rounded
2015-08-21-110758_1920x1080_scrot

Card Groups are not correctly un-rounded in v4. If there is nobody working on it yet, I can take a stab at it.

### PR DESCRIPTION
**Pull request by azmenak**
Fixes #17203 

Looks much better when using groups without an image at the top

### CODE REVIEW

**Comment by hnrch02** [comment]
The question is, should we support card groups with only one card? Because then we would need something like `:not(:only-child)`. CC @mdo

File: scss/_card.scss [hunk 1]
```diff
@@ -241,6 +241,9 @@
     // Handle rounded corners
     @if $enable-rounded {
       &:first-child {
+        border-top-right-radius: 0;
+        border-bottom-right-radius: 0;
```
Comment (hnrch02) [inline scss/_card.scss]: You can use the [`border-<side>-radius` mixins](https://github.com/twbs/bootstrap/blob/69f92408630707fe704eae309a71867adefa3d39/scss/mixins/_border-radius.scss) here.

**Comment by azmenak** [comment]
@hnrch02 good call on the mixins, that's been updated.

Not sure what the thoughts are for single-card groups; but I tested changing the lines to:
```scss
&:not(:only-child) {
  @include border-right-radius(0);
}
```

and that seems to have the intended effect. Should we include that in this PR?

**Comment by hnrch02** [comment]
@azmenak I'd wait on a response from @mdo on that since he'd call if we support single-card groups.

**Comment by cvrebert** [comment]
@mdo Ping

**Comment by mdo** [comment]
Added in 078c7ca because of merge conflicts.

**Pull request closed without merging**